ethpm/utils/registry.py

```diff
@@ -0,0 +1,48 @@
+from urllib import parse
+
+from typing import List
+
+from ethpm.exceptions import UriNotSupportedError
+
+from ethpm.validation import (
+    validate_package_name,
+    validate_registry_uri_authority,
+    validate_registry_uri_scheme,
+    validate_registry_uri_version,
+)
+
+
+def is_ens_domain(auth: str) -> bool:
```

Comment:
maybe go ahead and use the full name `authority`, otherwise the assumption is likely to be this stands for `authorization`